### PR TITLE
Bump is2view from 0.0.5 to 0.0.8

### PR DESCRIPTION
**Pull request by weiji14**
Bumps [is2view](https://github.com/tsutterley/IS2view) from 0.0.5 to 0.0.8.
- [Release notes](https://github.com/tsutterley/IS2view/releases)
- [Changelog](https://github.com/tsutterley/IS2view/blob/0.0.8/doc/source/release_notes/release-v0.0.8.rst)
- [Commits](https://github.com/tsutterley/IS2view/compare/0.0.5...0.0.8)

### CODE REVIEW

File: conda-linux-64.lock [hunk 1]
```diff
@@ -104,546 +112,620 @@ https://conda.anaconda.org/conda-forge/linux-64/xorg-xf86vidmodeproto-2.3.1-h7f9
...
+https://conda.anaconda.org/conda-forge/linux-64/wrapt-1.16.0-py311h459d7ec_0.conda#6669b5529d206c1f880b642cdd17ae05
 https://conda.anaconda.org/conda-forge/linux-64/xcb-util-image-0.4.0-h8ee46fc_1.conda#9d7bcddf49cbf727730af10e71022c73
-https://conda.anaconda.org/conda-forge/linux-64/xkeyboard-config-2.40-hd590300_0.conda#07c15d846a2e4d673da22cbd85fdb6d2
+https://conda.anaconda.org/conda-forge/linux-64/xkeyboard-config-2.41-hd590300_0.conda#81f740407b45e3f9047b3174fa94eb9e
 https://conda.anaconda.org/conda-forge/linux-64/xorg-libxext-1.3.4-h0b41bf4_2.conda#82b6df12252e6f32402b96dacc656fec
 https://conda.anaconda.org/conda-forge/linux-64/xorg-libxfixes-5.0.3-h7f98852_1004.tar.bz2#e9a21aa4d5e3e5f1aed71e8cefd46b6a
 https://conda.anaconda.org/conda-forge/linux-64/xorg-libxrender-0.9.11-hd590300_0.conda#ed67c36f215b310412b2af935bf3e530
-https://conda.anaconda.org/conda-forge/noarch/xyzservices-2023.10.1-pyhd8ed1ab_0.conda#1e0d85c0e2fef9539218da185b285f54
+https://conda.anaconda.org/conda-forge/noarch/xyzservices-2024.4.0-pyhd8ed1ab_0.conda#93dffc47dadbe36a1a644f3f50d4979d
```
Comment (weiji14) [inline conda-linux-64.lock:375]: @tsutterley, `xyzservices` is also updated here to `2024.4.0`, should fix your issue with the NASA GIBS urls.

Comment (tsutterley) [reply conda-linux-64.lock:375]: thanks @weiji14! you're the best! :tada:

**Review by tsnow03 — APPROVED** [review]

**Pull request merged**